Running in wrapper mode for '259'

Running python job.

[name: "CPU-QUOTA"
int_val: 19
int_val: 84
int_val: 7
int_val: 98
int_val: 6
int_val: 90
int_val: 79
, name: "DISK"
int_val: 32
int_val: 39
int_val: 33
int_val: 21
int_val: 11
int_val: 28
int_val: 16
, name: "MEMORY"
int_val: 20
int_val: 17
int_val: 19
int_val: 12
int_val: 37
int_val: 29
int_val: 11
, name: "NET"
int_val: 30
int_val: 22
int_val: 38
int_val: 17
int_val: 7
int_val: 13
int_val: 13
]
The paramters are {u'CPU-QUOTA': array([19, 84,  7, 98,  6, 90, 79]), u'NET': array([30, 22, 38, 17,  7, 13, 13]), u'DISK': array([32, 39, 33, 21, 11, 28, 16]), u'MEMORY': array([20, 17, 19, 12, 37, 29, 11])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 210000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 112640.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 32715571.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 637500000.0
Recovering the memory utilization
Setting cpu quota to 47500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 153600.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 26843545.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 750000000.0
Recovering the memory utilization
Setting cpu quota to 17500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 194560.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 27682406.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 712500000.0
Recovering the memory utilization
Setting cpu quota to 47500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 153600.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 26843545.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 750000000.0
Recovering the memory utilization
Setting cpu quota to 17500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 194560.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 27682406.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 712500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384a0cb0>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.44 seconds.
